server/events/vcs/gitlab_client_test.go

```diff
@@ -816,6 +816,10 @@ func TestGitlabClient_PullIsMergeable(t *testing.T) {
 						case fmt.Sprintf("/api/v4/projects/runatlantis%%2Fatlantis/merge_requests/%v", defaultMr):
 							w.WriteHeader(http.StatusOK)
 							w.Write(pipelineSuccess) // nolint: errcheck
+						case fmt.Sprintf("/api/v4/projects/runatlantis%%2Fatlantis/merge_requests/%v/approvals", c.mrID):
```

Comment:
Can you enhance the tests for this? It needs a test where `approvals_left` is `0` and another where `approvals_left` is greater than `0`.